src/stores/theme/createThemeStore.tsx

```
@@ -1,67 +1,263 @@
-          return {
-            configTheme: state.configTheme,
-            widgetTheme: state.widgetTheme,
-          };
+
+          Object.values(newStore.configThemeStates).forEach((themeState) => {
+            if (typeof themeState.expirationDate === 'string') {
+              themeState.expirationDate = new Date(themeState.expirationDate);
+            }
+          });
+
+          newStore.configTheme = newStore.configTheme;
```

Biome (2.1.2)

[error] configTheme is assigned to itself.

This is where is assigned.

Self assignments have no effect and can be removed.

(lint/correctness/noSelfAssign)